chapters/ch19.md

```diff
@@ -1,28 +1,27 @@
 # 19. Takeaways: Basic error analysis
 ->
-
+# 19. Điều cần nhớ: Phân tích lỗi cơ bản
 
 * When you start a new project, especially if it is in an area in which you are not an expert, it is hard to correctly guess the most promising directions.
 ->
-
+* Khi bạn bắt đầu một dự án mới, đặc biệt nếu đó là một lĩnh vực mà bạn không phải là chuyên gia, nó rất khó để dự đoán được những hướng giải quyết triển vọng nhất.
 
 * So don’t start off trying to design and build the perfect system. Instead build and train a basic system as quickly as possible—perhaps in a few days. Then use error analysis to help you identify the most promising directions and iteratively improve your algorithm from there.
 ->
-
+* Vì vậy đừng bắt đầu với việc thiết kế và xậy dựng một hệ thống hoàn hảo. Thay vào đó hãy xây dựng và huấn luyện một hệ thống cơ bản một cách nhanh nhất có thể-thậm chí chỉ trong một vài ngày. Sau đó, sử dụng phân tích lỗi để giúp bạn xác định những hướng triển vọng và lặp đi lặp lại cải thiện thuật toán của bạn từ đó.
 
 * Carry out error analysis by manually examining ~100 dev set examples the algorithm misclassifies and counting the major categories of errors. Use this information to prioritize what types of errors to work on fixing.
 ->
-
+* Thực hiện phân tích lỗi bằng cách kiểm tra thủ công khoảng 100 mẫu trong tập phát triển mà thuật toán phân loại sai và đếm những loại lỗi chính. Sử dụng thông tin này để ưu tiên các loại lỗi để khắc phục.
```

Comment:
> Sử dụng thông tin này để _sắp xếp thứ tự_ ưu tiên _khắc phục_ các loại lỗi ~~để khắc phục~~.

Reply:
Mình đề xuất từ "categories of errors" thành "những **hạng mục** lỗi" để đồng thuận với các chương trước. Mình cũng đề xuất cách dịch câu đó là
"... và điểm qua những hạng mục lỗi chính"